model/feature_flags.go

```diff
@@ -27,6 +31,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.TestBoolFeature = false
 	f.CloudDelinquentEmailJobsEnabled = false
 	f.CollapsedThreads = false
+	f.CustomUserStatuses = true
```

Comment:
this should be false by default

Reply:
@reflog this feature should be enabled by default for all users and servers

Reply:
We might need to talk to Cloud (or whoever owns feature flags) about that then since the intent was to have these features off by default and then gradually rolled out to Cloud servers before the flag finally gets removed and they're available in on-prem. On-prem users can still bypass the feature flag by setting an environment variable, but it wouldn't be on for them by default until the flag is removed.

Reply:
The gradual roll-out to cloud makes sense, I just want to make sure that when it hits on prem, it's enabled by default

Reply:
@esethna - this is opposite to current practices as outlined by @crspeller .
Until feature is deemed 'complete' the feature flag by default should be 'false'.

Reply:
You are correct, feature flags should be off by default. When ready to release to on-prem, they should be removed.